### ISSUE TITLE
"The Master Worker Encountered An Error: Zero Blocks" But There Are Blocks

### ISSUE THREAD

**Issue by TrillyReign** [openbullet/openbullet#748]
I keep receiving a pop-up that says " the master worker encountered an error: the config has zero blocks", whenever I paste loliscript code, via the loliscript view option (when making a config file)

This only happens if I paste the code. If I take the time to write everything out, or use stack view, then it works no issue. 
I had a 15 block config, that wouldn't work, simply because I copied, and pasted, one line :(

I am using V1.2.2. and it's not modded, or anything like that :+1: 

If you need any other information be sure to let me know!

**Comment by openbullet** [comment]
Hey, can you try to re-select the config in the runner? If you don't have live config editing enabled in the settings, whenever you make a change you need to re-select it. Also open the .loli with notepad and make sure there is content in it (after you paste the loliscript, you need to save. CTRL + S also works).

**Comment by openbullet** [comment]
If that doesn't help, maybe try to reproduce the issue with a sample config and post it here so I can replicate it on my end thanks.

**Comment by Fennzo** [comment]
I'm getting the same issue as well. Tries saving it and it didn't work.

**Comment by TrillyReign** [comment]
> Hey, can you try to re-select the config in the runner? If you don't have live config editing enabled in the settings, whenever you make a change you need to re-select it. Also open the .loli with notepad and make sure there is content in it (after you paste the loliscript, you need to save. CTRL + S also works).

Yes I did. This was my first response, as I thought turning off live editing, or re-selecting the config, would fix the issue. I then checked it in Notepad++ and it was BLANK. Didn't save one line of my config. 

I just tried CTRL+S and it still didn't work.

> If that doesn't help, maybe try to reproduce the issue with a sample config and post it here so I can replicate it on my end thanks.

I can, however, as I said, it isn't saving my configs due to this, so any config I send will be blank. 

As I said before, I noticed it only happens if I paste any lines in loliscript view. If this doesn't reproduce the issue for you, then I can take a screenshot and send log data
